examples/light-switch-app/nrfconnect/main/BindingHandler.cpp

```diff
@@ -0,0 +1,292 @@
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#include "BindingHandler.h"
+#ifdef CONFIG_CHIP_LIB_SHELL
+#include "ShellCommands.h"
+#endif
+
+#include <logging/log.h>
+LOG_MODULE_DECLARE(app);
+
+using namespace chip;
+using namespace chip::app;
+
+void BindingHandler::Init()
+{
+#ifdef CONFIG_CHIP_LIB_SHELL
+    SwitchCommands::RegisterSwitchCommands();
+#endif
+    chip::DeviceLayer::PlatformMgr().ScheduleWork(InitInternal);
+}
+
+void BindingHandler::OnOffProcessCommandUnicast(CommandId commandId, const EmberBindingTableEntry & binding, DeviceProxy * device,
+                                                void * context)
+{
+    CHIP_ERROR ret = CHIP_NO_ERROR;
+
+    auto onSuccess = [](const ConcreteCommandPath & commandPath, const StatusIB & status, const auto & dataResponse) {
+        ChipLogProgress(NotSpecified, "Binding command applied successfully!");
+    };
+
+    auto onFailure = [](CHIP_ERROR error) {
+        ChipLogError(NotSpecified, "Binding command was not applied! Reason: %" CHIP_ERROR_FORMAT, error.Format());
+    };
+
+    switch (commandId)
+    {
+    case Clusters::OnOff::Commands::Toggle::Id:
+        Clusters::OnOff::Commands::Toggle::Type toggleCommand;
+        ret = Controller::InvokeCommandRequest(device->GetExchangeManager(), device->GetSecureSession().Value(), binding.remote,
+                                               toggleCommand, onSuccess, onFailure);
+        break;
+
+    case Clusters::OnOff::Commands::On::Id:
+        Clusters::OnOff::Commands::On::Type onCommand;
+        ret = Controller::InvokeCommandRequest(device->GetExchangeManager(), device->GetSecureSession().Value(), binding.remote,
+                                               onCommand, onSuccess, onFailure);
+        break;
+
+    case Clusters::OnOff::Commands::Off::Id:
+        Clusters::OnOff::Commands::Off::Type offCommand;
+        ret = Controller::InvokeCommandRequest(device->GetExchangeManager(), device->GetSecureSession().Value(), binding.remote,
+                                               offCommand, onSuccess, onFailure);
+        break;
+    default:
+        ChipLogError(NotSpecified, "Invalid binding command data - commandId is not supported");
+        break;
+    }
+    if (CHIP_NO_ERROR != ret)
+    {
+        LOG_ERR("Invoke Unicast Command Request ERROR: %s", chip::ErrorStr(ret));
+    }
+}
+
+void BindingHandler::OnOffProcessCommandGroup(CommandId commandId, const EmberBindingTableEntry & binding, void * context)
+{
+    LOG_INF("Group OnOff Process Binding command...");
```

Comment:
I believe that many of logs like this in this file could be in the detailed and not in the info scope.